tron/core/job_collection.py

```diff
@@ -66,7 +69,13 @@ def update(self, new_job_scheduler):
         job_scheduler.schedule_reconfigured()
         return True
 
-    def restore_state(self, job_state_data, config_action_runner):
+    def restore_state(self, job_state_data, config_action_runner, tron_start_time=None):
+        # we will start looping through the jobs and their runs and restore
+        # state for each run in for a job and as we restore the state
+        # we will also schedule the next runs for that job
```

Comment:
this could probably be this functions docstring :)

```suggestion
        """
        Loops through the jobs and their runs in order to restore
        state for each run. As we restore state, we will also schedule the next 
        runs for each job
        """
```